data/en/cfmodule.json

```diff
@@ -2,6 +2,7 @@
 	"name":"cfmodule",
 	"type":"tag",
 	"syntax":"<cfmodule>",
+	"script":"cfmodule(template=\"nameoftemplate.cfm\",attributeCollection{});",
```

Comment:
Isn't `attributeCollection` supposed to be followed by `=`?

Reply:
Yup, that's what I get for copying from adobe website.